src/components/Dashboard/SortList/SortList.jsx

```diff
@@ -0,0 +1,134 @@
+import React, { Component } from 'react';
+import {
+  Dropdown,
+  ButtonGroup,
+  Button,
+} from 'react-bootstrap';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import * as sortListActions from '../../../actions/sortListActions';
+import './SortList.scss';
+import { ReactComponent as SortIcon } from './sort.svg';
+import { ReactComponent as SortIconAsc } from './sort-asc.svg';
+import { ReactComponent as SortIconDesc } from './sort-desc.svg';
+
+class SortList extends Component {
+  state = {};
+
+  constructor(props) {
+    super(props);
+    const { storageKey, sortListActions: { loadSort } } = this.props;
+
+    loadSort(storageKey);
+  }
+
+  onClickCallback = (newField, newDirection) => {
+    const { storageKey, allSortData, sortListActions: { setSort } } = this.props;
```

Comment:
`allSortData` -- what kind of data is this? The name is too vague for an outside reviewer to understand.

Reply:
allSortData - it stores sort settings for all dashboards and all columns.

Reply:
I don't want you to explain it. Your variable names should be clear enough without any explanation.

Reply:
I don't explain. I just show that variable name shows the sense of its content.

Reply:
It doesn't, David, or I wouldn't be asking. Think how to change the variable so it is self-explanatory